src/schemas/api-reference.json

```diff
@@ -7,7 +7,7 @@
   "properties": {
     "version": {
       "description":          "API reference version",
-      "enum":                 [0, "0.2.0"]
+      "enum":                 [0]
```

Comment:
I'm not sure what the version is for, if we always keep it at 0, also when it is in production. :/